src/kitBlocking.ts

```diff
@@ -460,17 +460,23 @@ export default class KitBlocker {
         if (!this.blockUserAttributes) {
             return false
         }
-        if (this.blockUserAttributes) {
-            const matchedAttributes = this.dataPlanMatchLookups['user_attributes'];
-            if (matchedAttributes === true) {
-                return false
-            }
-            if (!matchedAttributes[key]) {
-                return true
+        const matchedAttributes = this.dataPlanMatchLookups['user_attributes'];
+
+        // When additionalProperties is set to true, matchedAttributes 
+        // will be a boolean, otherwise it will return an object
+        if (typeof matchedAttributes === 'boolean' && matchedAttributes) {
```

Comment:
it's also unclear to me what combination of settings will have `additionalAttributes === true`, and so `matchedAttributes` is a `boolean`. can you share a data plan that results in this?  i've only ever seen `additionalAttributes === false` when `Block unplanned user attributes` is enabled, and `Allow unplanned attributes` is selected

Reply:
nevermind!  seems to be a caching issue.  i ended up downloading the plans instead to avoid having to wait for cache to bust.